test/e2e/BUILD

```diff
@@ -218,18 +219,14 @@ go_library(
     ],
 )
 
-# this is a private library until we can support building the
-# test binary
-# TODO(mikedanese): fix @io_bazelbuild/rules_go
-go_library(
+go_test(
     name = "go_default_test",
     srcs = [
         "e2e_test.go",
         "metrics_grabber_test.go",
     ],
     library = ":go_default_library",
     tags = ["automanaged"],
```

Comment:
I can add the "integration" tag which we use elsewhere to skip these types of tests. automanagement is still nice for integration tests.